src/field.rs

```diff
@@ -829,7 +843,7 @@ mod tests {
         assert_eq!(two * two, four);
         assert_eq!(two * one, two);
         assert_eq!(two * zero, zero);
-        assert_eq!(one * F::from(int_modulus), zero);
+        assert_eq!(one * zero, zero);
```

Comment:
This doesn't seem equivalent to the test that was deleted. Is the `one * F::modulus == zero` case not interesting anymore?

Reply:
`F::try_from(int_modulus)` would return `FieldError::ModulusOverflow`, so I can't translate it literally. I figured this would be the next best thing.